aws/acm.go

```diff
@@ -125,7 +125,7 @@ func (cc *certificateCache) listCerts() ([]*acm.CertificateSummary, error) {
 		for _, cert := range page.CertificateSummaryList {
 			certList = append(certList, cert)
 		}
-		return true // never stop iterator
+		return page.NextToken != nil
```

Comment:
This is duplicated with https://github.com/aws/aws-sdk-go/blob/master/aws/request/request_pagination.go#L75